src/index.ts

```diff
@@ -245,11 +265,7 @@ export = function plugin(
         ); // @note do not `catch` here - let error propagate to rollup level.
     },
 
-    generateBundle(
-      generateOptions: { file?: string },
-      bundle: { [fileName: string]: RollupAssetInfo | RollupChunkInfo },
-      isWrite: boolean,
-    ): Promise<any> {
+    generateBundle(generateOptions, _, isWrite) {
```

Comment:
this whole block is inside the `return` statement of the function which has an explicit return type `import('rollup').Plugin`  so type are inferred from there

Reply:
Right though when on a small screen we don't see that the type is inferred so makes code reading/grokking more difficult - let's keep them.

Reply:
These functions are implementing rollup hooks so I think they should infer parameter from the original type.
Before my changes the harcoded types where wrong (`Promise<any>` is not assignable to `Promise<void>`, which is the correct return) and to solve the issue a casting to entire entire object was added below:
https://github.com/elycruz/rollup-plugin-sass/blob/ee6f5e1f851540a9563f9a5392e48dfc27950900/src/index.ts#L287

Removing the incorrect types allows also to remove this casting which was hiding the wrong implementation. 

If the problem is see the type of parameters someone can just hover on the variable.  
Besides that this will be inconsistent  with all the other function of the plugin which haven't explicit types.

As an alternative I can inline all functions with explicit types coming from `rollup` and then return the plugin object, instead of creating the functions directly when returning the object.

Reply:
> As an alternative I can inline all functions with explicit types coming from rollup and then return the plugin object, instead of creating the functions directly when returning the object.

Actually this would lead with major conflicts with the modern compiler PR so I'm restoring the types so we can move on